src/Nut/DebugRoutes.php

```diff
@@ -23,9 +24,9 @@ protected function configure()
         $this
             ->setName('debug:routes')
             ->setDescription('System route debug dumper.')
-            ->addOption('sort-bind', null, InputOption::VALUE_NONE, 'Sort in order of bind name (default).')
+            ->addOption('sort-route', null, InputOption::VALUE_NONE, 'Sort in order of route name (default).')
             ->addOption('sort-pattern', null, InputOption::VALUE_NONE, 'Sort in order of URI patterns.')
-            ->addOption('sort-methods', null, InputOption::VALUE_NONE, 'Sort in order of HTTP method grouping allowed.')
+            ->addOption('sort-method', null, InputOption::VALUE_NONE, 'Sort in order of HTTP method grouping allowed.')
```

Comment:
This is where I was saying it seems to make more sense to have a value here instead of an option for each sorting. `--sort name/path/method`.